crates/stateless/src/fork_spec.rs

```diff
@@ -64,7 +65,9 @@ pub enum ForkSpec {
 
 impl From<ForkSpec> for ChainSpec {
     fn from(fork_spec: ForkSpec) -> Self {
-        let spec_builder = ChainSpecBuilder::default();
+        // We initialize with empty genesis since we only use hardforks from the constructed ChainSpec.
+        let spec_builder =
+            ChainSpecBuilder::default().genesis(Genesis::default()).chain(Chain::mainnet());
```

Comment:
The `.genesis` and `.chain` are required simply because if not provided ChainSpecBuilder panics -- we aren't interested in those fields anyway -- clarified this in a comment.